src/WinUI/ArcGIS.WinUI.Viewer/Samples/Data/ManageFeatures/ManageFeatures.xaml

```diff
@@ -0,0 +1,25 @@
+<UserControl 
+    x:Class="ArcGIS.WinUI.Samples.ManageFeatures.ManageFeatures"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime"
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
+  <Grid>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource BorderStyle}">
+      <StackPanel>
+        <TextBox Text="Select an option." />
+        <ComboBox x:Name="MethodPicker"
+                  IsEditable="False"
+                  IsEnabled="True"
+                  SelectionChanged="OnComboBoxSelectionChanged" />
+        <TextBox x:Name="InstructionLabel"
+                 Text="Select an option to show instructions."
+                 FontWeight="SemiBold" />
+        <ComboBox x:Name="DamageTypeDropDown"
```

Comment:
`DamageTypeDropDown` should be hidden when edit functionality is not being used.